web/template/admin/admin_tabs/server-notifications.gohtml

```diff
@@ -1,7 +1,6 @@
 {{define "server-notifications-admin-list"}}
 <link rel="stylesheet" href="/static/node_modules/flatpickr/dist/flatpickr.min.css">
 <script src="/static/node_modules/flatpickr/dist/flatpickr.min.js"></script>
-<script defer async src="/static/assets/ts-dist/server-notifications.js"></script>
```

Comment:
Such scripts are now bundled into main.js (this particular one is removed though)